src/components/timeline/TimelineContextMenu.svelte

```diff
@@ -235,7 +238,7 @@
   }
 </script>
 
-<ContextMenu hideAfterClick on:hide bind:this={contextMenuComponent}>
+<ContextMenu hideAfterClick={false} on:hide bind:this={contextMenuComponent}>
```

Comment:
This can be removed now right? Otherwise we'd need to manually close the context menu after actions are performed.

Reply:
Ugh! Oops 🤦🏻 . Removed!